README.md

```diff
@@ -36,6 +36,32 @@ It will install WireGuard (kernel module and tools) on the server, configure it,
 
 Run the script again to add or remove clients!
 
+## Configure clients
+
+### RHEL 8
+
+* Install the necessary packages:
+
+```
+sudo dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm https://www.elrepo.org/elrepo-release-8.el8.elrepo.noarch.rpm
+sudo dnf install dnf-plugins-core
+sudo dnf config-manager --enable elrepo
+sudo dnf install kmod-wireguard wireguard-tools
+```
+
+* Create file `/etc/wireguard/wg0.conf` with contents of the client config generated by the script (copy paste from server).
+
+* Connect: `systemctl start wg-quick@wg0.service`
+
+* Verify your public IP has changed to the server's IP, e.g.: `curl https://ident.me`
```

Comment:
```suggestion
* Verify your public IP has changed to the server's IP, e.g.: `curl ipv4.icanhazip.com ; curl ipv6.icanhazip.com`
```
Ident.me is blocked by pihole, so if a pihole dns is used, the command won't work. It does work with the default DNS servers. This solution will show both ipv4 and ipv6 addresses.